File: c++/0155-min-stack/main.cpp
#include <cstdlib>
#include <iostream>

#include "min_stack.cpp"

using namespace std;

int main() {
    MinStack* ms = new MinStack();
    ms->push(-2);
    ms->push(0);
    ms->push(-3);
    cout << "-3: " << ms->getMin() << endl;
    ms->pop();
    cout << "0: " << ms->top() << endl;
    cout << "-2: " << ms->getMin() << endl;
    delete ms;

    return EXIT_SUCCESS;
}

File: java/0155-min-stack/Main.java
public class Main {
    public static void main(String[] args) {
        MinStack minStack = new MinStack();
        minStack.push(-2);
        minStack.push(0);
        minStack.push(-3);
        System.out.println("-3: " + minStack.getMin());
        minStack.pop();
        System.out.println("0: " + minStack.top());
        System.out.println("-2: " + minStack.getMin());
    }
}
